src/main/java/lotto/modal/LottoRank.java

```diff
@@ -0,0 +1,43 @@
+package lotto.modal;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+
+	NOTHING_RANK(0, 0),
+	FOURTH_RANK(3, 5000),
+	THIRD_RANK(4, 50000),
+	SECOND_RANK(5, 1500000),
+	FIRST_RANK(6, 2000000000);
```

Comment:
```suggestion
	FIRST_RANK(6, 2_000_000_000);
```

위와 같이 가독성을 키울 수 있어요.